consensus/consensus_v2.go

```diff
@@ -264,13 +264,12 @@ func (consensus *Consensus) Start(
 					}
 					if k != timeoutViewChange {
 						consensus.getLogger().Warn().Msg("[ConsensusMainLoop] Ops Consensus Timeout!!!")
-						viewID := consensus.GetCurBlockViewID()
-						consensus.startViewChange(viewID + 1)
+						consensus.SetViewChangingID(consensus.GetCurBlockViewID())
```

Comment:
why set SetViewChangingID here? It will be set inside startViewChange() right?

Reply:
This is for fallback mode, in case of get view id failed from blockchain.  Yes it will be set after successfully getnextviewid withing startviewchange func.  The fallback mode will use the current viewchanging id to get the next one, which is current + 1, that's the old algorithm.  that's why it is the fallback mode.